### PR TITLE
TINY-8564: Fix double border with toolbar_location: bottom

### PR DESCRIPTION
**Pull request by noxuhax**
Related Ticket: TINY-8564

Description of Changes:
* Fix double border with `toolbar_location: bottom`

Pre-checks:
* [x] Changelog entry added
* [x] ~Tests have been added (if applicable)~
* [x] Branch prefixed with `feature/` for new features (if applicable)

Review:
* [x] Milestone set
* [x] Review comments resolved

### CODE REVIEW

File: modules/oxide/src/less/skins/ui/tinymce-5/skin.less [hunk 1]
```diff
@@ -111,8 +116,14 @@
     background: @toolbar-background !important;
   }
 
+  .tox-menubar + .tox-toolbar,
```
Comment (lnewson) [inline modules/oxide/src/less/skins/ui/tinymce-5/skin.less:119]: This rule will never match, as you have the same rule below that will always override it.

File: modules/tinymce/CHANGELOG.md [hunk 2]
```diff
@@ -11,6 +11,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Fixed a regression whereby text patterns couldn't be updated at runtime #TINY-8540
 - Naked buttons better adapt to various background colors, improved text contrast in notifications #TINY-8533
 - The autocompleter would not fire the `AutocompleterStart` event nor close the menu in some cases #TINY-8552
+- Fixed double border with `toolbar_location: bottom` #TINY-8564
```
Comment (lnewson) [inline modules/tinymce/CHANGELOG.md:14]:
```suggestion
- Fixed a double border showing when using `toolbar_location: 'bottom'` #TINY-8564
```

**Review by lnewson — APPROVED** [review]
Pulled it down and compared the tinymce-5 skin to TinyMCE 5 and it looked the same to me 👍

File: modules/oxide/src/less/skins/ui/tinymce-5/skin.less (hunk 1 above)
Comment (noxuhax) [reply modules/oxide/src/less/skins/ui/tinymce-5/skin.less]: Great eye! Fixed

**Review by ltrouton — APPROVED** [review]

**Pull request merged**